src/game.cpp

```diff
@@ -2819,6 +2835,45 @@ void Game::toggleHud(float *statustext_time, bool *flag)
 		client->setHighlighted(client->getHighlighted(), *flag);
 }
 
+void Game::toggleMinimap(float *statustext_time, bool *flag, bool *show_hud)
+{
+	if (*show_hud && g_settings->getBool("enable_minimap")) {
+		MinimapMode mode = mapper->getMinimapMode();
+		mode = static_cast<MinimapMode>(static_cast<int>(mode) + 1);
```

Comment:
You shouldn't use C++ style casts unless they're necessary.  In any case, are you sure you can't simply increment mode?  It doesn't seem to give any warnings for me